deployment/template/endpoints.yaml.tmpl

```diff
@@ -21,7 +21,7 @@ config_version: 3
 name: "<SERVICE_NAME>"
 
 # API title to appear in the user interface (Google Cloud Console).
-title: Data Commons API (Testing)
+title: <API_TITLE>
```

Comment:
sorry, i don't follow where this string gets replaced (or what the final title would be). would it now also be $service_name?

Reply:
It would be replaced in https://github.com/datacommonsorg/mixer/pull/395/files#diff-1306a0330ffe59d264ca80671199aee3b508d384954c97adf36ca8374f8e58d6R41

Currently the. title looks like below. It's 1) not an API (exposed externally) and 2) not test..
![image](https://user-images.githubusercontent.com/5951856/102398807-9ad08a80-3f94-11eb-803d-d2f5dcd4f75d.png)

Reply:
gotcha.

since it will be replaced with service_name, would it be clearer to do:
title: <SERVICE_NAME>

i also don't see name: getting updated in the script.

Reply:
good catch! 

I also realized this change is in the "mixer" repo, so it would be used by the standalone mixer as well. In that case, this title would be something like "Data Commons API - Staging(Prod)". And this title should be set in the configuration. It's still written as env.sh, which I add a todo to move to yaml file.

"website" repo import this file as sub-module. Once this PR is submitted, I will change the part in "website" repo